tests/system/fixtures/cli-login.expect

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env expect -f
```

Comment:
love it. haven't seen `expect` scripts in quite some time (since we used them to configure Cisco IOS routers in a previous life).